torchvision/transforms/functional.py

```diff
@@ -728,14 +728,20 @@ def _get_inverse_affine_matrix(center, angle, translate, scale, shear):
     # Thus, the inverse is M^-1 = C * RSS^-1 * C^-1 * T^-1
 
     angle = math.radians(angle)
-    shear = math.radians(shear)
+    print(shear)
+    if isinstance(shear, (tuple, list)) and len(shear) == 2:
```

Comment:
What if the length of `shear` is `> 2`, or `== 1`? This will go into the next `else` block, and will fail.

Maybe we could add an assert before this part of the code to assert those constraints?

Reply:
I've added an `else` statement to handle these errors and raise an exception, as I wanted to avoid code duplication (checking the length twice: in the assert and again to create the shear list).
What do you think?

Reply:
This looks ok to me!